src/content-handlers/iiif/modules/uv-shared-module/css/right-panel.less

```diff
@@ -122,4 +122,4 @@
             display: block;
         }
     }
-}
+}
```

Comment:
Only change to this file is loss of end-of-file line break. Can we put it back to avoid unnecessary diffs?